src/wp-includes/link-template.php

```diff
@@ -1592,21 +1592,33 @@ function get_delete_post_link( $post = 0, $deprecated = '', $force_delete = fals
 }
 
 /**
- * Retrieves the edit comment link.
- *
- * @since 2.3.0
- *
- * @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
- * @return string|void The edit comment link URL for the given comment.
- */
-function get_edit_comment_link( $comment_id = 0 ) {
+* Retrieves the edit comment link.
+*
+* @since 2.3.0
+*
+* @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
+* @param string         $context    Optional. Context in which the URL should be used. Either 'display',
+* to include HTML entities, or 'display'. Default 'display'.
+* @return string|void The edit comment link URL for the given comment.
+*/
```

Comment:
```suggestion
 * Retrieves the edit comment link.
 *
 * @since 2.3.0
 *
 * @param int|WP_Comment $comment_id Optional. Comment ID or WP_Comment object.
 * @param string         $context    Optional. Context in which the URL should be used. Either 'display',
 *                                   to include HTML entities, or 'display'. Default 'display'.
 * @return string|void The edit comment link URL for the given comment.
 */
```